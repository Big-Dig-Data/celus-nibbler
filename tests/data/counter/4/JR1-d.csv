Journal Report 1 (R4),Number of Successful Full-Text Download Requests by Month and Journal,,,,,,,,,,,,,,,,,,,,
MyOrg,,,,,,,,,,,,,,,,,,,,,
,,,,,,,,,,,,,,,,,,,,,
Period covered by Report:,,,,,,,,,,,,,,,,,,,,,
2022-01-01 to 2022-12-31,,,,,,,,,,,,,,,,,,,,,
Date run:,,,,,,,,,,,,,,,,,,,,,
1/1/2023,,,,,,,,,,,,,,,,,,,,,
Journal,Publisher,Platform,Journal DOI,Proprietary Identifier,Print ISSN,Online ISSN,Reporting Period Total,Reporting Period HTML,Reporting Period PDF,22-Jan,22-Feb,22-Mar,22-Apr,22-May,22-Jun,22-Jul,22-Aug,22-Sep,22-Oct,22-Nov,22-Dec
Journal1,,,,,,,147277,,,10042,15922,15708,15796,8560,8481,7853,8446,14887,15002,16902,9678
