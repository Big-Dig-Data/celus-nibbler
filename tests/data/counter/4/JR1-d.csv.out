"2022-01-01","2022-01-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","10042"
"2022-02-01","2022-02-28","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","15922"
"2022-03-01","2022-03-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","15708"
"2022-04-01","2022-04-30","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","15796"
"2022-05-01","2022-05-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","8560"
"2022-06-01","2022-06-30","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","8481"
"2022-07-01","2022-07-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","7853"
"2022-08-01","2022-08-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","8446"
"2022-09-01","2022-09-30","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","14887"
"2022-10-01","2022-10-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","15002"
"2022-11-01","2022-11-30","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","16902"
"2022-12-01","2022-12-31","","Journal1","","","","","","Platform:|Publisher:","FT Article Requests","9678"
